ring/util_test.go

```diff
@@ -71,6 +71,14 @@ func (r *RingMock) GetTokenRangesForInstance(_ string) (TokenRanges, error) {
 	return []uint32{0, math.MaxUint32}, nil
 }
 
+func (r *RingMock) InstancesInZoneCount(zone string) int {
```

Comment:
Just to make the linter happy.

```suggestion
func (r *RingMock) InstancesInZoneCount(_ string) int {
```